common/lib/xmodule/xmodule/randomize_module.py

```diff
@@ -61,13 +59,19 @@ def __init__(self, *args, **kwargs):
         if self.choice is not None:
             self.child_descriptor = self.descriptor.get_children()[self.choice]
             # Now get_children() should return a list with one element
-            log.debug("children of randomize module (should be only 1): %s",
-                      self.get_children())
-            self.child = self.get_children()[0]
+            self.child = self.get_chsildren()
```

Comment:
Also, rather than setting `self.child_descriptor` and `self.child` here in `__init__`, can you make `@properties` for both of those, and then have `get_children` and `get_child_descriptors` use `self.choice` directly?